chartfx-chart/src/main/java/io/fair_acc/chartfx/axes/spi/format/DefaultLogFormatter.java

```diff
@@ -12,6 +12,7 @@
 /**
  * @author rstein
  */
+@Deprecated // supported by DefaultFormatter
```

Comment:
IMO its safe to delete it if its functionality is subsumed in the DefaultFormatter. The only usecase would be as an example to implement custom Formatters. If there is some feature of the old implementation that users rely on it's still possible to incorporate the old version and reintroduce it if needed or add the functionality to your new Formatter.